src/components/billingRecord/IFXBillingRecordList.vue

```diff
@@ -271,11 +272,14 @@ export default {
           if (!value && value !== false) continue
           // TODO: make this check more generalized for multiple item types
           // Check for different item types
-          if (header.value.toLowerCase().includes('date')) {
+          const header_value = header.value.toLowerCase()
+          if (header_value === 'startdate' || header_value === 'enddate') {
```

Comment:
Since you're matching the exact text, why do you want to lowercase the `value`? Is this for other `date` matches?